traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -650,7 +650,7 @@ sub profile_ds_data {
 				my $re = $host_re;
 				$re =~ s/\\//g;
 				$re =~ s/\.\*//g;
-				my $hname    = $ds_type =~ /^DNS/ ? "edge" : "ccr";
+				my $hname    = $ds_type =~ /^DNS/ ? "edge" : "http_routing";
```

Comment:
As a heads up, once #865 merges, almost every line here will have a merge conflict because it replaces the hardcoded `"edge"` prefix with the delivery service's new `routing_name` column. So when resolving the conflict make sure to keep those changes as well as your `"http_routing"` changes. Both changes will work together.